app/scripts/app.js

```diff
@@ -404,6 +409,9 @@ angular
       .when('/project/:project/browse/deployments-replicationcontrollers/:rc', {
         redirectTo: '/project/:project/browse/rc/:rc'
       })
+      .when('/project/:project/browse/pet-sets/:petset', {
+        redirectTo: '/project/:project/browse/stateful-sets/:statefulset'
```

Comment:
Not anymore, but initially i was thinking we would get this in to support PetSets, then update to StatefulSets. Will remove.